samcli/commands/check/calculation.py

```diff
@@ -7,17 +7,19 @@
 
 
 from samcli.commands._utils.resources import AWS_LAMBDA_FUNCTION
-from samcli.commands.check.resources.Graph import Graph
-from .resources.Warning import CheckWarning
+from samcli.commands.check.resources.graph import CheckGraph
+from .resources.warning import CheckWarning
 
 
-class Calculation:
-    _graph: Graph
+class CheckCalculation:
+    _graph: CheckGraph
 
-    def __init__(self, graph: Graph):
+    def __init__(self, graph: CheckGraph):
         """
-        Args:
-            graph (Graph): The graph object. This is where all of the data is stored
+        Parameters
+        ----------
+            graph: Graph
```

Comment:
Should this be `CheckGraph` as well?

Reply:
I missed that one. I'll double check all of those docs and make any necessary changes.